service/controller/deleter.go

```diff
@@ -0,0 +1,122 @@
+package controller
+
+import (
+	"time"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/certs"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/controller"
+	"github.com/giantswarm/operatorkit/informer"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/kvm-operator/service/controller/v13"
+)
+
+type DeleterConfig struct {
+	CertsSearcher certs.Interface
+	G8sClient     versioned.Interface
+	K8sClient     kubernetes.Interface
+	Logger        micrologger.Logger
+
+	ProjectName string
+}
+
+type Deleter struct {
+	*controller.Controller
+}
+
+func NewDeleter(config DeleterConfig) (*Deleter, error) {
```

Comment:
This is the new deleter controller which does the job of the node controller which we remove now.